crates/spicepod/src/spec.rs

```diff
@@ -8,6 +8,7 @@ use crate::component::{dataset::Dataset, ComponentOrReference};
 #[serde(rename_all = "lowercase")]
 pub enum SpicepodVersion {
     V1Beta1,
+    V1Beta2,
```

Comment:
Why a new version?

Reply:
I assumed V1Beta1 referred to the very first iteration. Reverted.